src/casp/services/api/data_manager/bigquery_response_parsers.py

```diff
@@ -2,27 +2,29 @@
 
 from google.cloud import bigquery
 
+from casp.services.api import schemas
 
-def _parse_match_object(row: bigquery.Row) -> t.Dict[str, t.Any]:
+
+def _parse_match_object(row: bigquery.Row) -> schemas.AnnotationInfoCellTypeCount:
```

Comment:
We might want to rename the module too if it's not going to be BQ anymore

Reply:
At the moment it still does parses the bigqeury responses. Once we rewrite CellTypeCount based annotation, we will have to get rid of these files.

Reply:
Ah ok gotcha